packages/nimble-components/src/table-column/enum-text/cell-view/index.ts

```diff
@@ -0,0 +1,65 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { styles } from '../../text-base/cell-view/styles';
+import { template } from '../../text-base/cell-view/template';
+import type {
+    TableColumnEnumCellRecord,
+    TableColumnEnumColumnConfig
+} from '../../enum-base';
+import type { ConvertedKeyMappingText } from '../../../mapping/text';
+import { TableColumnTextCellViewBase } from '../../text-base/cell-view';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-column-enum-text-cell-view': TableColumnEnumTextCellView;
+    }
+}
+
+/**
+ * A cell view for displaying mapped text
+ */
+export class TableColumnEnumTextCellView extends TableColumnTextCellViewBase<
+TableColumnEnumCellRecord,
+TableColumnEnumColumnConfig
+> {
+    public override get text(): string {
+        return this.getMappingToRender()?.label ?? '';
```

Comment:
Is there a reason we decided to display empty text instead of allowing clients to set a placeholder at the column level?

I guess it feels a bit redundant to offer both a placeholder and a default mapping. But one difference between them is that the placeholder would render with grey text to indicate the value is "invalid". If there isn't strong rationale for omitting placeholder, I'd vote to include it for consistency with other text column types.

Reply:
I guess it makes sense to have it for consistency. I'll add it.